receiver/haproxyreceiver/README.md

```diff
@@ -1,11 +1,16 @@
 # HAProxy Receiver
 
+<!-- status autogenerated section -->
 | Status                   |           |
 | ------------------------ |-----------|
 | Stability                | [alpha]   |
 | Supported pipeline types | metrics   |
 | Distributions            | [contrib] |
 
+[alpha]: https://github.com/open-telemetry/opentelemetry-collector#alpha
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+<!-- end autogenerated section -->
```

Comment:
I was planning to do something similar for metrics. We can put a list of collected metrics here with links to more detailed docs for each.